src/index.js

```diff
@@ -0,0 +1,145 @@
+"use strict";
+
+const state = {
+    city: "Seattle",
+    tempCount: 0
+}
+
+let temperature = document.getElementById("temp-counter__span");
+
+// Function to set sky
+const skyImg = { 
+    "0" : "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+    "1": "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+    "2" : "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+    "3" : "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+};
```

Comment:
Numbers as keys work fine but I think it would be as easy and more readable to call them "sunny", "cloudy", etc-- 😉